File: OP/lab4/lab4 Python/TFunction.py
class TFunction:
    def __init__(self, a, b, c=0):
        self._a = a
        self._b = b
        self._c = c

    def increase_by(self, num):
        self._a += num
        self._b += num
        self._c += num

    def decrease_by(self, num):
        self._a -= num
        self._b -= num
        self._c -= num

    def print(self):
        pass

    def calculate(self, x):
        pass

File: OP/lab4/lab4 C++/lab4 C++/TFunction.cpp
#include "TFunction.h"

TFunction::TFunction(double a, double b, double c)
{
    this->a = a;
    this->b = b;
    this->c = c;
}

void TFunction::increaseBy(double num)
{
    a += num;
    b += num;
    c += num;
}

void TFunction::decreaseBy(double num)
{
    a -= num;
    b -= num;
    c -= num;
}
